beacon-chain/rpc/beacon_server.go

```diff
@@ -125,9 +137,100 @@ func (bs *BeaconServer) DomainData(ctx context.Context, request *pb.DomainReques
 // state.latest_eth1_data is updated, and validator deposits up to this root can be processed.
 // The deposit root can be calculated by calling the get_deposit_root() function of
 // the deposit contract using the post-state of the block hash.
-//
-// TODO(#2307): Refactor for v0.6.
 func (bs *BeaconServer) Eth1Data(ctx context.Context, _ *ptypes.Empty) (*pb.Eth1DataResponse, error) {
```

Comment:
This doesn't need to be exported right? 

```suggestion
func (bs *BeaconServer) eth1Data(ctx context.Context, _ *ptypes.Empty) (*pb.Eth1DataResponse, error) {
```